src/conversions.jl

```diff
@@ -209,9 +209,10 @@ if VERSION < v"0.4.0-dev+4319" # prior to 0.4 tuple-type changes
         if len != length(tt)
             throw(BoundsError())
         end
-        ntuple(len, i ->
-               convert(tt[i], PyObject(ccall((@pysym :PySequence_GetItem), PyPtr,
-                                             (PyPtr, Int), o, i-1))))
+        ntuple((i ->
```

Comment:
Ahh. I was a little bit paranoid after getting many errors on `f(@compat sth, sth-else)` not working. I can remove this....